pkg/ops/miningManager.go

```diff
@@ -77,12 +78,17 @@ func CreateMiningManager(
 	account *rpc.Account,
 ) (*MiningMgr, error) {
 
-	group, err := pow.SetupMiningGroup(cfg, exitCh)
+	powFilterLogger, err := logging.ApplyFilter(*cfg, ComponentName, logger)
```

Comment:
here and few other places
```suggestion
	logger, err := logging.ApplyFilter(*cfg, ComponentName, logger)
```

Reply:
problem with that is that we user logger twice, to make both powFilterLogger and opsFilterLogger. So redefining the variable will mess up the creation of opsFilterLogger.

Reply:
Aah I see the issue. I think we don't actually need filtering for the sub-component for now lets apply log filtering only for the `MiningMgr`